source/blog/2016-04-20-scala-210-and-java7.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Final Scala 2.10 and Java 7 release
+published: true
+post_author:
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer, Ostrich
+---
+
+The upcoming releases of our family of projects will be the final release for
+Scala 2.10 and Java 7. These should be Finagle 6.35.0, Finatra 2.2.0,
+Util 6.34.0, Scrooge 4.7.0, TwitterServer 1.20.0, and Ostrich 9.18.0.
+
+The reason for this is because Java 7 hit it's
+[end-of-life](http://www.oracle.com/technetwork/java/eol-135779.html)
+back in April 2015. We would like to begin using the Java 8 libraries in
+our code and remove some Java 7 specific workarounds. Once Scala 2.12 is
+released we will work on adding 2.11/2.12 cross-compile support.
```

Comment:
Suggested verbiage change:
Once Scala 2.12 is released, we plan to add support for cross-compiling Scala 2.11 and 2.12.